tests/controller_tests/test_download.py

```diff
@@ -94,3 +98,46 @@ def test_sort_columns_exception(test_session):
     assert str(e.value) == (
         "No table sort order defined for table extract: UnspecifiedSheet. Please add " "sheet to TABLE_SORT_ORDERS"
     )
+
+
+def test_get_find_download_file_metadata(test_session, test_buckets):
+    filename = "example.xlsx"
+    filebytes = b"example file contents"
+    file = FileStorage(io.BytesIO(filebytes), filename=filename, content_type=EXCEL_MIMETYPE)
+
+    upload_file(file=file, bucket=Config.AWS_S3_BUCKET_FIND_DATA_FILES, object_name=filename)
+
+    response = test_session.get(f"/get-find-download-metadata/{filename}")
+    assert response.status_code == 200
+    assert "content_length" in response.json
+    assert "content_type" in response.json
+    assert "last_modified" in response.json
```

Comment:
Can we please move these to `integration_tests`, since we're testing actual communication to/from S3 (localstack)?